README.adoc

```diff
@@ -134,6 +134,15 @@ Next set the URL of the Gradle Enterprise instance to which the build scans shou
 
 image::enable-auto-injection.png[Enable auto-injection,{thumbnail}]
 
+It is also possible to enable injection for specific repositories by specifying VCS repository patterns. These are comma-separated values, which will be used in a contains check against the repository URL.
+
+image::general-git-vcs-repository-filtering.png[Git VCS repository filtering, {thumbnail}]
+
```

Comment:
Hm...interesting thought...The original idea was to have it comma-separated in case lots of values are specified + it makes parsing predictable. 

@guylabs Any thoughts on this from your side? Since it's in beta anyway, we can also reevaluate and change it later I guess, right?

Reply:
Yes I'd keep it for now and we can improve later. We may as well consider this when we have the exclude filters then we can write something like:

```
+:foo-bar*
-:beta
```

Similar what TeamCity does with branch filters.

Reply:
Exactly.

One this to note is that if we do some major rework in terms of how filter looks like or is stored, users will have to write it again, as old data is going to be erased (I'm not sure there is a way to migrate the configuration actually...and perhaps it's not worth it given the feature is in beta anyway)